stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -296,6 +300,7 @@ internal class ChatDomainImpl internal constructor(
             setOfflineEnabled(offlineEnabled)
         }.build()
 
+        latestUsers = repos.observeLastUsers()
```

Comment:
Should we rename it to `observeLatestUsers` to make it consistent? Or maybe we can rename it to `cachedUsers`. What do you think?